fulvio-almeida/1.i/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
As this document is in portuguese language we need to update the attribute `lang` to "pt-br"